en/messages.json

```diff
@@ -311,5 +311,45 @@
   },
   "tooltipWarning": {
     "message": "This Container is set up to use a Mozilla VPN proxy, but Mozilla VPN is not on. Turn Mozilla VPN on to use this Container."
+  },
+  "getMozillaVpn": {
+    "message": "Get Mozilla VPN"
+  },
+  "integratewithmozillavpn": {
+    "message": "Integrate your Containers with Mozilla VPN"
+  },
+  "mozillaVpnAndProxyPermissionsTitle": {
+    "message": "Mozilla VPN & Proxy Permissions",
+    "description": "Section headline in settings"
+  },
+  "nativeMessagingPermissionTitle": {
+    "message": "Allow extension to exchange messages with programs other than Firefox",
+    "description": "Permission title"
+  },
+  "nativeMessagingPermissionDescription": {
+    "message": "This setting allows the extension to exchange messages with the Mozilla VPN client and is required for Mozilla VPN integration features to work.",
+    "description": "'This setting' refers to the permission indicated in nativeMessagingPermissionTitle"
+  },
+  "proxyPermissionTitle": {
+    "message": "Allow Firefox to control proxy settings",
```

Comment:
I'm confused: I don't think it's Firefox, but the extension, that needs to be able to control proxy settings?

Reply:
It's confusing but this mimics the language used by the browser to describe this permission.